cms/envs/common.py

```diff
@@ -41,12 +41,8 @@
 from xmodule.modulestore.edit_info import EditInfoMixin
 
 ############################ FEATURE CONFIGURATION #############################
+STUDIO_NAME = "Studio"
```

Comment:
Supporting i18n for a brand name will, in most cases, require multiple different translation strings, so supporting i18n on this is not straightforward.